pallets/fragments/src/lib.rs

```diff
@@ -183,11 +183,11 @@ pub struct UniqueOptions {
 
 /// **Struct** of a **Fragment Definition**
 #[derive(Encode, Decode, Clone, scale_info::TypeInfo, Debug, PartialEq)]
-pub struct FragmentDefinition<TFungibleAsset, TAccountId, TBlockNum> {
+pub struct FragmentDefinition<TU8Array, TFungibleAsset, TAccountId, TBlockNum> {
```

Comment:
Also why?

Reply:
> Also why?

reason give above (basically because `DefinitionMetadata` is a parameter of one of the extrinsics)